store/tikv/gcworker/gc_worker.go

```diff
@@ -512,15 +434,15 @@ func (w *GCWorker) deleteRanges(ctx goctx.Context, safePoint uint64) error {
 	return nil
 }
 
-func resolveLocks(ctx goctx.Context, store *tikvStore, safePoint uint64, identifier string) error {
+func resolveLocks(ctx goctx.Context, store tikv.Storage, safePoint uint64, identifier string) error {
```

Comment:
Seems we can make it a method of `tikvStore` and put it inside tikv package.

Reply:
We can do it in future PR.